components/generic/PageHeader.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <header class="row">
+    <b-col
+      cols="12"
+      :lg="mediaUrl ? 9 : 12"
+      class="col lead mt-3"
+    >
+      <h1
+        data-qa="gallery title"
+      >
+        {{ title }}
+      </h1>
+      <!-- eslint-disable vue/no-v-html -->
+      <div
+        v-if="htmlDescription"
+        v-html="htmlDescription"
+      />
+      <!-- eslint-enable vue/no-v-html -->
+      <p v-if="rawDescription">
+        {{ rawDescription }}
+      </p>
```

Comment:
Does it break when you have just 1 description and use only

`<div
        v-if="htmlDescription"
        v-html="htmlDescription"
      />`
?

Reply:
We end up with a missing <p> tag if we just use htmldescription.